UPGRADE-1.13.md

```diff
@@ -904,3 +904,26 @@ and use one of the new attributes accordingly to the type of your class, e.g.:
 
 1. Check if the `countryCode` is set has been removed from the `Sylius\Component\Addressing\ModelAddress::setProvinceCode` 
    method. The adequate check is ensured by validation.
+
+1. The migration changing the type of the `DC2TYPE:array` to `JSON` has been added to the following fields in tables:
+    - `data` => `sylius_address_log_entries`
+    - `roles` => `sylius_admin_user`
+    - `configuration` => `sylius_catalog_promotion_action`
+    - `configuration` => `sylius_catalog_promotion_scope`
+    - `configuration` => `sylius_product_attribute`
+    - `configuration` => `sylius_promotion_action`
+    - `configuration` => `sylius_promotion_rule`
+    - `configuration` => `sylius_shipping_method`
+    - `configuration` => `sylius_shipping_method_rule`
+    - `roles` => `sylius_shop_user`
+
+   If you don't want this change and prefer skipping this migration you can do it by executing the following commands:
```

Comment:
```suggestion
   If you prefer skipping this migration and applying the changes yourself you can do it by executing the following commands:
```
Guessing outright skipping the migration and doing nothing will blow up since the metadata and database will be out of sync.